### PR TITLE
Add/fix more stubs

### PR DESCRIPTION
**Pull request by nvnieuwk**
## PR checklist

Closes #XXX 

- [ ] This comment contains a description of changes (with reason).
- [ ] If you've fixed a bug or added code that should be tested, add tests!
- [ ] If you've added a new tool - have you followed the module conventions in the [contribution docs](https://github.com/nf-core/modules/tree/master/.github/CONTRIBUTING.md)
- [ ] If necessary, include test data in your PR.
- [ ] Remove all TODO statements.
- [ ] Emit the `versions.yml` file.
- [ ] Follow the naming conventions.
- [ ] Follow the parameters requirements.
- [ ] Follow the input/output options guidelines.
- [ ] Add a resource `label`
- [ ] Use BioConda and BioContainers if possible to fulfil software requirements.
- Ensure that the test works with either Docker / Singularity. Conda CI tests can be quite flaky:
  - For modules:
    - [ ] `nf-core modules test <MODULE> --profile docker`
    - [ ] `nf-core modules test <MODULE> --profile singularity`
    - [ ] `nf-core modules test <MODULE> --profile conda`
  - For subworkflows:
    - [ ] `nf-core subworkflows test <SUBWORKFLOW> --profile docker`
    - [ ] `nf-core subworkflows test <SUBWORKFLOW> --profile singularity`
    - [ ] `nf-core subworkflows test <SUBWORKFLOW> --profile conda`

### CODE REVIEW

File: modules/nf-core/bcftools/sort/tests/main.nf.test [hunk 1]
```diff
@@ -9,7 +9,28 @@ nextflow_process {
...
+            assertAll(
+                { assert process.success },
+                { assert snapshot(process.out).match("vcf") }
+            )
+        }
+
+    }
+
+    test("sarscov2 - vcf - stub") {
         when {
```
Comment (SPPearce) [inline modules/nf-core/bcftools/sort/tests/main.nf.test:34]:
```suggestion
        options "-stub"
        when {
```
Here, I think you dropped this.

Comment (nvnieuwk) [reply modules/nf-core/bcftools/sort/tests/main.nf.test:34]: Thanks!

File: modules/nf-core/bcftools/concat/tests/main.nf.test [hunk 2]
```diff
@@ -97,7 +97,7 @@ nextflow_process {
             assertAll(
                 { assert process.success },
                 { assert snapshot(
-                    file(process.out.vcf[0][1]).name,
+                    process.out.vcf,
```
Comment (SPPearce) [inline modules/nf-core/bcftools/concat/tests/main.nf.test:100]: This is now checking everything, so just process.out?

Comment (nvnieuwk) [reply modules/nf-core/bcftools/concat/tests/main.nf.test:100]: Yeah true, but it's the same now right? :p I'll change that if the tests are failing now (don't want to overdo the CI right now :grin:)

**Review by SPPearce — APPROVED** [review]
Happy once the CI is.

**Pull request merged**